sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/data/cosmos/CosmosDataAutoConfiguration.java

```diff
@@ -1,55 +1,55 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-
-package com.azure.spring.cloud.autoconfigure.data.cosmos;
-
-import com.azure.spring.cloud.autoconfigure.implementation.cosmos.properties.AzureCosmosProperties;
-import com.azure.spring.data.cosmos.config.AbstractCosmosConfiguration;
-import com.azure.spring.data.cosmos.config.CosmosConfig;
-import com.azure.spring.data.cosmos.core.CosmosTemplate;
-import com.azure.spring.data.cosmos.core.ResponseDiagnosticsProcessor;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
-import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
-import org.springframework.boot.autoconfigure.condition.ConditionalOnExpression;
-import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
-import org.springframework.context.annotation.Configuration;
-
-/**
- *  {@link EnableAutoConfiguration Auto-configuration} for Spring Data Cosmos support.
- *
- *  @since 4.0.0
- */
-@Configuration(proxyBeanMethods = false)
-@ConditionalOnClass({ CosmosTemplate.class })
-@ConditionalOnExpression("${spring.cloud.azure.cosmos.enabled:true}")
-@ConditionalOnProperty(prefix = "spring.cloud.azure.cosmos", name = { "endpoint", "database" })
-public class CosmosDataAutoConfiguration extends AbstractCosmosConfiguration {
-
-    private final AzureCosmosProperties cosmosProperties;
-    private final ResponseDiagnosticsProcessor responseDiagnosticsProcessor;
-
-    CosmosDataAutoConfiguration(AzureCosmosProperties cosmosProperties,
-                                @Autowired(required = false) ResponseDiagnosticsProcessor responseDiagnosticsProcessor) {
-        this.cosmosProperties = cosmosProperties;
-        this.responseDiagnosticsProcessor = responseDiagnosticsProcessor;
-    }
-
-    @Override
-    protected String getDatabaseName() {
-        return cosmosProperties.getDatabase();
-    }
-
-    @Override
-    public CosmosConfig cosmosConfig() {
-        final CosmosConfig.CosmosConfigBuilder builder = CosmosConfig.builder();
-        builder.enableQueryMetrics(cosmosProperties.isPopulateQueryMetrics());
-
-        if (responseDiagnosticsProcessor != null) {
-            builder.responseDiagnosticsProcessor(responseDiagnosticsProcessor);
-        }
-
-        return builder.build();
-    }
-
-}
+//
```

Comment:
Can this file compile? If it can, we don't need to comment it out, we can just remove it from the spring.factories?